dali/pipeline/executor/lowered_graph.h

```diff
@@ -467,8 +469,9 @@ class DLL_PUBLIC OpGraph {
    */
   void RemoveOpNode(OpNodeId id);
 
-  std::map<std::string, TensorNodeId> tensor_name_to_id_;
+  std::map<std::string, TensorNodeId, std::less<>> tensor_name_to_id_;
```

Comment:
Passing `std::less<>` allows use of any comparable key for `find` and `operator[]` - in this case, a `string_view`.